Formula/litani.rb

```diff
@@ -0,0 +1,70 @@
+class Litani < Formula
+  include Language::Python::Virtualenv
+
+  desc "Metabuild system"
+  homepage "https://awslabs.github.io/aws-build-accumulator/"
+  url "https://github.com/awslabs/aws-build-accumulator/archive/refs/tags/1.19.0.tar.gz"
+  sha256 "1b15b076e4af7c1bd057d52a7b7c9f4101f9d40581f16f089ad6b9863b35b4fa"
+  license "Apache-2.0"
+
+  depends_on "coreutils" => :build
+  depends_on "mandoc" => :build
+  depends_on "scdoc" => :build
+  depends_on "gnuplot"
+  depends_on "graphviz"
+  depends_on "ninja"
+  depends_on "python@3.9"
+
+  resource "Jinja2" do
+    url "https://files.pythonhosted.org/packages/39/11/8076571afd97303dfeb6e466f27187ca4970918d4b36d5326725514d3ed3/Jinja2-3.0.1.tar.gz"
+    sha256 "703f484b47a6af502e743c9122595cc812b0271f661722403114f71a79d0f5a4"
+  end
+
+  resource "MarkupSafe" do
+    url "https://files.pythonhosted.org/packages/bf/10/ff66fea6d1788c458663a84d88787bae15d45daa16f6b3ef33322a51fc7e/MarkupSafe-2.0.1.tar.gz"
+    sha256 "594c67807fb16238b30c44bdf74f36c02cdf22d1c8cda91ef8a0ed8dabf5620a"
+  end
+
+  resource "voluptuous" do
+    url "https://files.pythonhosted.org/packages/c0/2c/ccbeb25364e3e0c5e4522f13d66e2fc639bb4d4ecdf73be0959552cbecb4/voluptuous-0.12.2.tar.gz"
+    sha256 "4db1ac5079db9249820d49c891cb4660a6f8cae350491210abce741fabf56513"
+  end
+
+  resource "PyYAML" do
+    url "https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz"
+    sha256 "607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e"
+  end
+
+  def install
+    ENV.prepend_path "PATH", libexec/"vendor/bin"
+    venv = virtualenv_create(libexec/"vendor", Formula["python@3.9"].opt_bin/"python3")
```

Comment:
```suggestion
    venv = virtualenv_create(libexec/"vendor", "python3")
```
Do we really need to be explicit about `python@3.9` here? This makes changing the `python@3.x` dependency harder later on, since it's easy to forget to change these references.

Reply:
Resolved